src/preact/component/3p-frame.js

```diff
@@ -105,7 +104,7 @@ function ProxyIframeEmbedWithRef(
     };
     setNameAndSrc({
       name: JSON.stringify({
-        'host': parseUrlDeprecated(src).hostname,
+        'host': new URL(src).hostname,
```

Comment:
This only works if `src` is an absolute URL, where `parseUrlDeprecated` will handle relative URLs.

Reply:
Does passing `location` as the 2nd arg work? If so...Done.